schematic/manifest/generator.py

```diff
@@ -1552,6 +1544,25 @@ def create_manifests(jsonld:str, data_types:list, access_token:Optional[str]=Non
         Returns:
             Union[List[str], List[pd.DataFrame], BinaryIO]: a list of Googlesheet URLs, a list of pandas dataframes or an Excel file.
         """
```

Comment:
You should remove the BinaryIo type from the union. As far as I can tell when you pick excel its' returning a dataframe.

Reply:
No, when you pick Excel, it returns an excel file.

Reply:
Thanks for your feedback Andrew! I think if you pick "excel", the code would return a list of Excel file paths. I will correct the typing.

Reply:
Should this be `excel file path` or can you get multiple excel sheets from this?

Reply:
I think this should be excel file "paths" (instead of "path") because the current behavior is that the code would still return multiple excel file paths. The part about "returning excel sheet to the user" has moved to the API side (see routes.py).. and even if multiple excel file paths get returned by this function, only one file would get returned to users in routes.py